flask/app/error_handler.py

```diff
@@ -1,11 +1,21 @@
-from flask import Blueprint, jsonify, current_app as app
+from flask import Blueprint, json, jsonify, request, current_app as app
 from werkzeug.exceptions import HTTPException
 import traceback
 from sqlalchemy import exc
 from .extensions import db
 
 error_blueprint = Blueprint("error_handler", __name__)
 
+# dump relavant request info into a string
+def get_request_info() -> str:
+    info: str = "\n"
+    info += f"{str(request)}\n"
+    info += f"Query Params:\n{json.dumps(request.args.to_dict(), indent=4)}\n"
+    if request.method != "GET":
+        body = request.get_data(parse_form_data=True)
+        info += f"Body:\n{json.dumps(json.loads(body), indent=4)}\n"
```

Comment:
We are reading the entire HTTP payload and treating it as JSON conditional only on the request method not being GET! Payloads could be attached to GET requests too (unconventional), but more importantly, if the payload is not meant to be JSON or is badly formed, this will error, and we cannot have an error handler raise an exception itself.